EXPT_NAME theanolm-100k
EXPT_PARAMS expt10
EXPT_WORK_DIR /scratch/work/moisioa3/conv_lm/experiments/theanolm-100k/expt10
Experiment: theanolm-100k/expt10
Job ID: 54291647
Task ID: 
Host: ivb17.int.triton.aalto.fi
Start date: Fri Jun 26 19:09:41 EEST 2020
Work directory: /scratch/work/moisioa3/conv_lm/experiments/theanolm-100k/expt10
Temporary directory: /tmp/54291647

Currently Loaded Modules:
  1) anaconda3/latest   2) gcc/9.2.0

 

floatX=float32,device=cuda0,base_compiledir=/tmp/theano,exception_verbosity=high,openmp=False
WARNING (theano.tensor.blas): Using NumPy C-API based implementation for BLAS functions.
ERROR (theano.gpuarray): Could not initialize pygpu, support disabled
Traceback (most recent call last):
  File "/scratch/work/groszt1/envs/theanoLM/lib/python3.7/site-packages/theano/gpuarray/__init__.py", line 227, in <module>
    use(config.device)
  File "/scratch/work/groszt1/envs/theanoLM/lib/python3.7/site-packages/theano/gpuarray/__init__.py", line 214, in use
    init_dev(device, preallocate=preallocate)
  File "/scratch/work/groszt1/envs/theanoLM/lib/python3.7/site-packages/theano/gpuarray/__init__.py", line 99, in init_dev
    **args)
  File "pygpu/gpuarray.pyx", line 658, in pygpu.gpuarray.init
  File "pygpu/gpuarray.pyx", line 587, in pygpu.gpuarray.pygpu_init
pygpu.gpuarray.GpuArrayException: b'Could not load "libcuda.so": libcuda.so: cannot open shared object file: No such file or directory'
TheanoLM 1.3.2
Theano 1.0.4
pygpu 0.7.6
==
+ theanolm score /scratch/work/moisioa3/conv_lm/experiments/theanolm-100k/expt10/nnlm.h5 /scratch/work/moisioa3/conv_lm/data/devel/plain.txt --output-file /scratch/work/moisioa3/conv_lm/experiments/theanolm-100k/expt10/perplexity-devel.txt --output perplexity --exclude-unk --shortlist
WARNING (theano.tensor.blas): Using NumPy C-API based implementation for BLAS functions.
ERROR (theano.gpuarray): Could not initialize pygpu, support disabled
Traceback (most recent call last):
  File "/scratch/work/groszt1/envs/theanoLM/lib/python3.7/site-packages/theano/gpuarray/__init__.py", line 227, in <module>
    use(config.device)
  File "/scratch/work/groszt1/envs/theanoLM/lib/python3.7/site-packages/theano/gpuarray/__init__.py", line 214, in use
    init_dev(device, preallocate=preallocate)
  File "/scratch/work/groszt1/envs/theanoLM/lib/python3.7/site-packages/theano/gpuarray/__init__.py", line 99, in init_dev
    **args)
  File "pygpu/gpuarray.pyx", line 658, in pygpu.gpuarray.init
  File "pygpu/gpuarray.pyx", line 587, in pygpu.gpuarray.pygpu_init
pygpu.gpuarray.GpuArrayException: b'Could not load "libcuda.so": libcuda.so: cannot open shared object file: No such file or directory'
2020-06-26 19:09:52,589 get_default_device: Theano is not using a GPU or an old version of libgpuarray is installed.
2020-06-26 19:09:52,590 from_file: Reading vocabulary from network state.
2020-06-26 19:09:55,428 from_file: Number of words in vocabulary: 2427252
2020-06-26 19:09:55,429 from_file: Number of words in shortlist: 100003
2020-06-26 19:09:55,429 from_file: Number of word classes: 100003
2020-06-26 19:09:55,430 from_file: Building neural network.
slurmstepd: error: *** JOB 54291647 ON ivb17 CANCELLED AT 2020-06-26T19:10:12 ***
